BUILD/settings/any.dat

```diff
@@ -48,3 +48,4 @@ auto_forceFatLootToken	boolean	force grabbing the fat loot tokens from daily dun
 auto_pvpEnable	boolean	Break the hippy stone to unlock PvP?
 auto_towerBreak	string	Where should we break in the tower? (e.g. wall of skin, wall of bones, shadow, ns). Blank by default.
 auto_ignoreRestoreFailure	boolean	if true we will always ignore failure to restore MP or HP and just continue playing
+auto_useBatteries	boolean	If true will use harvest and use batteries during breakfast. If false will only harvest.
```

Comment:
The description here is misleading. It will always harvest regardless of what this says. This only affect whether it is used in combat. To be honest I would get rid of it, there is no reason to disable using it in combat and making it an opt in means most people will not even notice the feature is available.

Reply:
```suggestion
auto_useBatteries	boolean	If true will use batteries. Batteries are always harvested regardless of this variable
```